### PR TITLE
adding control panel support for mobile

### PR DESCRIPTION
**Pull request by carsonsuite**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea/*
```
Comment (RyanThompson) [inline .gitignore]: Can you remove this? Personal files (even though I have one too) need to be ignored in your global .gitignore file.

**Comment by RyanThompson** [comment]
Nice work! Just remove that `.idea/` stuff and Ill merge er in ^_^

**Comment by carsonsuite** [comment]
Cool I'll do that now

**Comment by RyanThompson** [comment]
You da man! Thanks!

Ryan Thompson
President / CEO / Streams Evangelist
PyroCMS, Inc.
www.pyrocms.com

t (877) 705-0806
m (309) 752-8581

**Comment by carsonsuite** [comment]
Okay that should fix it.. grrr Love/Hate PhpStrom mostly love

**Comment by RyanThompson** [comment]
NOICE! Thanks a ton Carson

**Pull request merged**